logger/Cargo.toml

```
@@ -0,0 +1,44 @@
+[package]
+name = "shuttle-logger"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[dependencies]
+async-broadcast = "0.5.1"
+async-trait = { workspace = true }
+chrono = { workspace = true }
+clap = { workspace = true }
+opentelemetry-proto = { workspace = true, features = ["gen-tonic", "logs", "traces"] }
```

```suggestion
opentelemetry-proto = { workspace = true, features = ["gen-tonic", "traces"] }
```
Since we are no longer capturing logs